src/Pokemon.java

```diff
@@ -10,7 +10,11 @@ public abstract class Pokemon {
     //Constructor
     //public Pokemon(){};
     //public Pokemon(String name, enum color, int healthPointLevel, String environment){};
-    public Pokemon(String name, int healthPointLevel, String environment){};
+    public Pokemon(String name, int healthPointLevel, String environment){
+        this.name = name;
+        this.healthPointLevel = healthPointLevel;
+        this.environment = environment;
+    };
 
     //getters
```

Comment:
Ik zie dat je niet meteen bent begonnen op een nieuwe branch, zodoende kan ik ook niet je subclasses inzien. Ik kan ook in deze klasse alleen bij regel 10 t/m 16 een comment achterlaten. 

Waar ik een comment over wil maken is jou fightsWith methode op regel 37:
De naam suggereert dat je hier ook je weapon gebruikt, bijvoorbeeld ("I can fight with " + this.weapon)